packages/core/package.json

```diff
@@ -90,10 +74,7 @@
   "author": "",
   "license": "MIT",
   "files": [
-    "src/",
-    "dist/",
-    "README.md",
-    "THEME_README.md"
```

Comment:
Only need to specify `dist`, when packing npm takes all the default files such as README, LICENSE etc. The `src` is not needed here